torch/fx/graph_module.py

```diff
@@ -390,12 +390,12 @@ def _gen_model_repr(module_name: str, module: torch.nn.Module) -> Optional[str]:
         for buffer_name, buffer in self._buffers.items():
             if buffer is None:
                 continue
-            model_str += f"{tab*2}self.register_buffer('{buffer_name}', torch.empty({list(buffer.shape)}))\n"
+            model_str += f"{tab*2}self.register_buffer('{buffer_name}', torch.empty({list(buffer.shape)}, dtype={buffer.dtype}))\n"
```

Comment:
Is this repr always guaranteed to be the valid python expression that refers to the dtype?

Reply:
As far as I can tell, yeah.

![image](https://user-images.githubusercontent.com/6355099/146292490-a0170878-1458-49d3-a8d3-a24697e21814.png)